setup.cfg

```
@@ -57,6 +57,8 @@ install_requires =
     pip-conflict-checker==0.6.*
     ipython>=7.13.0,<=7.20.0
     pimms
+    AFQ-Browser @ git+https://github.com/yeatmanlab/AFQ-Browser@83cc7261198b1a26b6ba07c323f94a2418cb9c82
```

Before we merge this, I should address https://github.com/yeatmanlab/AFQ-Browser/issues/289, so that we can pin a version from pypi here.

---

I think this can be updated to:

```suggestion
    AFQ-Browser>=0.3
```